assets/src/scss/blocks/PostsList/PostsListStyle.scss

```diff
@@ -208,17 +212,11 @@
     font-size: calc(var(--font-size-xxs--font-family-primary) - 2px);
     display: flex;
 
+    &,
```

Comment:
What is the purpose of this `&`?

Reply:
So it means to apply the same style to whatever element or text is found in that selector. Reason being that when the author override is used, the name is no longer a link so previously the styles didn't apply to the text that was overridden.